api/db/changelog/db.changelog-258-create-vwb-user-pod-table.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="tarekahmed" id="db.changelog-258-create-vwb-user-pod-table">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <tableExists tableName="vwb_user_pod"/>
+      </not>
+    </preConditions>
+    <createTable tableName="vwb_user_pod">
+      <column name="vwb_user_pod_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
+        <constraints nullable="false" foreignKeyName="fk_user_id" references="user(user_id)"/>
+      </column>
+      <column name="vwb_pod_id" type="varchar(100)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="is_active" type="boolean">
+        <constraints nullable="false"/>
```

Comment:
I think we want to track whether a pod is initial billing or BYO. is this the intern use of this column, if yes, is_active might not be acturate because it also means pod inactive in the lab. Maybe other name "is_initial_credit_active" or somthing else with "initial credit" indication

Reply:
sure, I'll update this.